src/igvfd/schemas/changelogs/differentiated_cell.md

```diff
@@ -0,0 +1,2 @@
+## Changelog for differentiated_cell.json
+* renamed in_vitro.json to differentiated_cell.json
```

Comment:
Try to follow this example: https://github.com/IGVF-DACC/igvfd/blob/IGVF-128-rm-parenthesis-temp-schema/src/igvfd/schemas/changelogs/treatment.md

Start with the title "Minor revision..." And write it in a sentence structure (caps your first word in the. sentence)

Reply:
You may use something on these lines: the original profile named "in_vitro" is now renamed to be "differentiated_cell"